core/src/com/unciv/ui/overviewscreen/EmpireOverviewCategories.kt

```diff
@@ -47,7 +48,10 @@ enum class EmpireOverviewCategories(
     Resources("StatIcons/Happiness", 'R', Align.topLeft,
         fun (viewingPlayer: Civilization, overviewScreen: EmpireOverviewScreen, persistedData: EmpireOverviewTabPersistableData?)
                 = ResourcesOverviewTab(viewingPlayer, overviewScreen, persistedData),
-        fun (viewingPlayer: Civilization) = viewingPlayer.detailedCivResources.isEmpty().toState()),
+        fun (viewingPlayer: Civilization) = when {
+            viewingPlayer.detailedCivResources.any { it.resource.resourceType != ResourceType.Bonus } -> EmpireOverviewTabState.Normal
+            else -> EmpireOverviewTabState.Disabled
```

Comment:
The addition of .any{...} looks good, but the if/else should be done with .toState(), like before

Reply:
Fixed